### ISSUE TITLE
License Validity

### ISSUE THREAD

**Issue by Silun** [Opticos/GWSL-Source#191]
Hello,

I have purchased and used X410 for many years. At some point, the dev just decided that everyone, including those who had already purchased the full version of X410, was going to need a subscription to use the app, effectively undoing the purchase.

Now, I'm willing to pay for good software, but I'm not willing to be (what feels like being) scammed like that. You list GWSL as free, but it is (in effect) paid-for software, no doubt about that. So my question is: What am I really buying when I purchase GWSL in the MS Store? For how long will I keep getting updates? Are you at all committed to not pull off what the X410 devs have done?

Long story short, I want to know whether the purchase of GWSL is _one-time_ or _recurring in one way or another_.

Thank you!
